app/server/api/transfer_account_api.py

```diff
@@ -119,7 +119,7 @@ def put(self, transfer_account_id):
             if not approve == transfer_account.is_approved and transfer_account.is_approved is not True:
                 transfer_account.approve_and_disburse()
 
-            db.session.commit()
+            db.session.flush()
```

Comment:
i mean i think there's a 90% we'll end up shifting all of these back pretty soon but ok lol